designsafe/static/scripts/data-depot/components/nees-publication/nees-publication.template.html

```diff
@@ -99,6 +99,32 @@ <h2>{{$ctrl.project.name}}: {{$ctrl.project.metadata.project.title}}</h2>
                         </td>
                     </tr>
 
+                    <tr ng-if="$ctrl.project.metadata.experiments[0].doi">
+                        <td style="padding:0px;">
+                            <table class="table" style="margin-bottom:0;">
+                                <tbody>
+                                    <tr>
+                                        <th style="border-top:none; width:25%">DOIs</th>
+                                        <td style="border-top:none; width:75%">
+                                            <button class="btn btn-sm btn-info" ng-click="$ctrl.showDoiList($ctrl.project)">
+                                                View
```

Comment:
For this "View" button and the other "Citation" button could you use this button here?:
![Screen Shot 2019-11-04 at 8 05 44 AM](https://user-images.githubusercontent.com/29575979/68126682-ef708d80-fed9-11e9-99c0-c2b2024fb16b.png)
It would be awesome if you could remove the style tags from the existing "citation" buttons and make them a new type of button in the CSS? They should only be in the templates shown at the top of the `published-view.component.js`